src/command.js

```diff
@@ -30,22 +31,23 @@ const Command = class {
   static create (rawInputs) {
     const command = doNotAllowMissingProperties(new this(rawInputs))
     this._sanitizeCreation(command)
-    this._validateCreation(command)
+    this._assertCreation(command)
 
     return command
   }
 
-  // If there are extra validations, put them here.
-  static _validateCreation (newlyCreatedCommandInstance) {
-    this.validateThatSchemaMustBeStatic(newlyCreatedCommandInstance)
+  // If there are extra assertions, put them here.
+  static _assertCreation (newlyCreatedCommandInstance) {
+    this._assertSchemaIsStatic(newlyCreatedCommandInstance)
```

Comment:
I think this indirection could be delayed until we actually have multiple things to assert which may never happen.

Reply:
I still like the fact that we have a `creation` section. And we can plug and play creation functions there.
But I will remove it, since you have more experience with our `Command` class.

`Also because of YAGNI.` 🌟 👍